runtime/gc_vlhgc/GlobalMarkCardScrubber.cpp

```diff
@@ -31,6 +31,7 @@
 #include "CallSitesIterator.hpp"
 #include "CardTable.hpp"
 #include "ClassLoaderClassesIterator.hpp"
+#include "ClassIterator.hpp"
 #include "ClassStaticsIterator.hpp"
```

Comment:
Here and everywhere it is applicable: remove includes for iterators are not used any more

Reply:
👍